ci/try

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+#
+# This script executes a given COMMAND until it succeeds or MAX_RETRIES have
+# been reached. In between retries the script waits for
+# (INITIAL_TIMEOUT)^RETRIES , such that the time between retries increases.
+
+MAX_RETRIES=${MAX_RETRIES:-5}
```

Comment:
I would suggest this to be 2 by default, IMHO 5 is too much retries and just set the var for jobs that tend to recover with more.

Reply:
Changed.